packages/cardano-services-client/src/HandleProvider/KoraLabsHandleProvider.ts

```diff
@@ -20,6 +20,7 @@ const paths = {
   handles: '/handles',
   healthCheck: '/health'
 };
+export type Handle = string;
```

Comment:
Can import this from the `core` package right?

Reply:
Fixed on 0938543